biblib/views/library_view.py

```diff
@@ -201,6 +235,7 @@ def solr_update_library(library_id, solr_docs):
                 # Update the database
                 library.bibcode = new_bibcode
                 session.add(library)
+                flag_modified(library, "bibcode")
```

Comment:
This is interesting - was the modification not getting pushed without this?

Reply:
Yeah. It has to do with the way the libraries model is structured and how we go about updating it compared to the other models we have in ADS. We get away with this in a lot of places because of our update functions rewriting entire rows of the DB, but the model updates in place and `postgres` isn't able to detect changes in nested `JSON` easily. https://github.com/sqlalchemy/sqlalchemy/discussions/6473